### PR TITLE
Adds token defaults

### LINKED ISSUE #75: Better tokens with default values
I'd like to see token be able to have a default value for when scripts are run non-interactivly or if the user is <kbd>return</kbd> happy.

I'm proposing that tokens be switched from `_{2}[A-Z0-9_]*_{2}` to  `\{{2}([A-Z0-9_]+)(:(.*))?\}{2}`. This would make tokens look like {{TOKEN:default}} or just {{TOKEN}}instead of __TOKEN__ and allow for handy default values in templates.

### PR DESCRIPTION
**Pull request by alecho**
Will close #75

### CODE REVIEW

**Comment by jtyost2** [comment]
Noted updated PR #67 and the README with the updated token format.

**Comment by beporter** [comment]
Looks good to me. :+1:

File: src/Console/FileParser.php [hunk 1]
```diff
@@ -96,7 +96,7 @@ public function parseTemplate($template, $config)
             if ($value === ' ') {
                 $value = '';
             }
-            $this->replaceInFile($token, $value, $template);
```
Comment (beporter) [inline src/Console/FileParser.php]: One more follow up question: Is it necessary to preserve this secret "enter a a space to blank the value" logic? If we want a value to default to the empty string, can't we use a token default value like this: `{{TOKEN:}}`?

Comment (alecho) [inline src/Console/FileParser.php]: I don't think so but I'll need to check.

**Pull request merged**